### ISSUE TITLE
Disabling IntelliSense/suggestions

### ISSUE THREAD

**Issue by daniahl** [microsoft/vscode#141904]
For accessibility reasons I have disabled things related to IntelliSense/suggestions globally in VSCode. I have the following in settings.json:
"editor.parameterHints.enabled": false,
 "editor.quickSuggestions": {
     "other": false,
     "comments": false,
     "strings": false
}

This doesn't seem to affect jupyter notebooks. It would be good to have the option, as IntelliSense can interfere with screen reader / magnifier software.

If there is currently a way to disable this please let me know. I wouldn't mind breaking things to get it done. :)

Running Windows 11, VSCode 1.63.2, vscode-jupyter v2022.1.1001760686

Thanks!

**Comment by greazer** [comment]
Jupyter extension doesn't manipulate when these intellisense features appear.

**Comment by daniahl** [comment]
Okay, so if this extension doesn't, which component does? I have intellisense disabled in a pure Python file, but not in an ipynb-notebook.

**Comment by blackeyeddan** [comment]
I ran into this today too and found `editor.suggestOnTriggerCharacters` to be the issue. If it is enabled, suggestions continue to show up as @daniahl reported.

For example, suggestions continue to appear given these settings:
```json
"editor.quickSuggestions": {
    "other": false,
    "comments": false,
    "strings": false
},
"editor.suggestOnTriggerCharacters": true
```

Suggestions DO NOT appear given these settings:
```json
"editor.quickSuggestions": {
    "other": false,
    "comments": false,
    "strings": false
},
"editor.suggestOnTriggerCharacters": false
```

It's also worth pointing out you can still press CTL+Space to see suggestions when desired, even with everything set to `false`.

**Comment by daniahl** [comment]
THANK YOU @blackeyeddan!!!
Now VSCode is usable for me, and I can press CTRL+Space when I wish the suggestions to appear.
Perfect!
